actionpack/test/dispatch/request_test.rb

```diff
@@ -1290,3 +1290,21 @@ def setup
     assert_equal expected_hints, early_hints
   end
 end
+
+class RequestInspectTest < BaseRequestTest
+  test "inspect" do
+    request = stub_request(
+      "REQUEST_METHOD" => "POST",
+      "REMOTE_ADDR" => "1.2.3.4",
+      "HTTP_X_FORWARDED_PROTO" => "https",
+      "HTTP_X_FORWARDED_HOST" => "glu.ttono.us:443",
```

Comment:
IMO, a [RFC 2606](https://tools.ietf.org/html/rfc2606) reserved name should be better:
```suggestion
      "HTTP_X_FORWARDED_HOST" => "example.org:443",
```

Reply:
Yes, good point. I've changed it to example.com. I had copied "glu.ttono.us" from another test 😄